README.md

```diff
@@ -1 +1,120 @@
-# leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Leaderboard</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [The leaderboard] <a name="about-project"></a>
+
+**[The leaderboard]** website displays scores submitted by different players
+
+# 🛠 Built With <a name="built-with">Technologies</a>
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+### Tech Stack <a name="tech-stack">HTML, CSS and JavaScript</a>
+
+The entire project was built with only HTML5, CSS3, JavaScript AND Webpack.
+
+# 📖 [To Do List] <a name="about-project"></a>
```

Comment:
### OPTIONAL

- [ ] Kindly delete the heading ` todo list` this is not relevant to this project 👍🏿

Reply:
Thanks, I have done it